### DISCUSSION TITLE
Discussions のクローズについて

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#331] [要望・アイデア・不具合]
自分の提案を聞き入れていただき改善が実現した場合、こちらからクローズの申請をする必要があるのでしょうか？　それともQiitaの判断で自動クローズとなるのでしょうか？

　例えば私の要望 #312 は既に解決していただきましたが、未だクローズされていません。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
適宜こちらでクローズしているので申請は不要です！
手動で対応しているため漏れてしまうこともありまして…🙇💦
https://github.com/increments/qiita-discussions/discussions/312 はクローズしておきました🙏

**Reply by tsubolabo** [reply]
了解しました。 #312 のクローズも確認できました。ありがとうございました。
